### DISCUSSION TITLE
Cambio sezione alunno da classe X a classe X

### DISCUSSION THREAD

**Discussion by luigirotondi** [iisgiua/giuaschool#793] [Richieste di aiuto] [answered]
Buon pomeriggio, desidero avere un chiarimento: quando un alunno cambia classe per motivi specifici, i voti assegnati nella sezione di provenienza vengono trasferiti alla nuova sezione? In tal caso, è previsto che tali voti vengano modificati dai docenti della nuova sezione?

**Comment by Davide031** [comment]
No, l'alunno perderà tutti i voti ma se lo rimetti nella classe dove lo hai tolto riappariranno.
Nella classe dove lui non appartiene non può avere voti

**Reply by luigirotondi** [reply]
Da norma l'alunno deve avere i voti della classe successiva.

**Comment by trinko** [comment]
Confermo, i vecchi voti non sono considerati nella nuova classe.
Però il coordinatore (o lo staff) possono vedere i vecchi voti dalla pagina SITUAZIONE.

Invece le assenze (numero totale di ore di assenza/ritardi/uscite) comprendono anche quelle fatte nella classe precedente.

**Reply by luigirotondi** [reply]
Non è possibile inserire la funzione richiesta da me nel senso che una volta che l'alunno cambia sezione i voti vengono almeno visualizzati da docenti?

**Reply by trinko** [reply]
> Non è possibile inserire la funzione richiesta da me nel senso che una volta che l'alunno cambia sezione i voti vengono almeno visualizzati da docenti?

Non è impossibile ma non è una priorità.

**Reply by luigirotondi** [reply]
Per caso possiamo inserire noi questa funzione o non è possibile?

**Reply by trinko** [reply]
> Per caso possiamo inserire noi questa funzione o non è possibile?

Certo, se la volete sviluppare voi non c'è problema.

**Reply by luigirotondi** [reply]
Quali file dovremmo maneggiare per aggiungere il codice?

**Reply by trinko** [reply, marked as answer]
> Quali file dovremmo maneggiare per aggiungere il codice?

Se vuoi fare apparire questi voti assieme agli altri:
- **src/Controller/VotiController.php**: per recuperare i dati anche degli altri voti; procedura **votiController()** per gestire il quadro dei voti.
- **templates/lezioni/voti_quadro.html.twig**: per modificare la visualizzazione nel quadro dei voti
